dephell_pythons/_finder.py

```diff
@@ -91,7 +96,10 @@ def get_version(self, path: Path) -> str:
         # get version from CLI for cpython
         if path.name.startswith('python'):
             # this works much faster, so let's do it if possible
-            result = subprocess.run([str(path), '--version'], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+            try:
+                result = subprocess.run([str(path), '--version'], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
What is the text of the exception you've caught here?

Reply:
I see it in description but I'm interested in the path that caused it.

Reply:
> What is the text of the exception you've caught here?

```
path = WindowsPath('C:/Workspace/Projects/dephell_pythons/tests/python3.7')
...
OSError: [WinError 193] %1 не является приложением Win32
```

Reply:
Hmmm. Looks like it's a wrong path in the first place.

Reply:
Currently, if the found file name startswith "python" it will be considered as python executable. `tests/python3.7` file is not a win32 executable file.

Reply:
https://github.com/dephell/dephell_pythons/blob/7dffed4975cd114df2d9474a578d6ff853b38978/dephell_pythons/_finder.py#L92

Reply:
Yes, of course, I know. The question is different:

1. Is this file even exist? What is it?
1. Is this path in the `PATH` (or what Win have instead, IDK)?